confirm-deposit

```diff
@@ -1 +1,109 @@
-# Title
+# Confirm a Deposit to the Exchange
+
+Deposits are made in several stages. 
+One of the stages is to go through the [Exchange server](https://docs.flare.network/exchange/architecture/#architecture-of-an-exchange), which monitors for deposits and updates the Balances database. 
+Run the code below to have the Exchange server listen for deposit transactions and confirm the deposits.
+
+On a blockchain, multiple [validators](h[ttps://docs.flare.network/tech/glossary/](https://docs.flare.network/tech/validators/)) examine newly created transactions and come to an agreement about whether they are valid. 
+When they are valid, they are added to a new block on the chain and identified by a one-way cryptographic algorithm, called a _hash_. 
+The one-way hash prevents transactions from being reverted. 
+[Blocks](https://docs.flare.network/tech/glossary/) are added to the [blockchain](https://docs.flare.network/tech/glossary/) in chronological order and it may take a few blocks for the hash to be completed.
+To prevent problems, we confirm deposits several blocks back (five blocks in our code), when the blocks are most likely to be irreversible.
+
+Newly submitted deposits to our receiving address are labeled "pending."
+After five blocks have been created, valid deposits are labeled "confirmed."
+
+Run the javascript below to see pending and confirmed transactions and which block number they were added to.
+
```

Comment:
TL;DR: I think we should have general education on the blockchain elsewhere, for example, in the glossary. Here are some terms to add to the glossary.

<!-- I don't think this article should be a primer on blockchain, so I limited blockchain explanation to only what they need to know for this procedure. 

| Term | Description |
| ----------- | ----------- |
| address | The address of a wallet where assets can be stored. Represented by a hash. |
| block header | Data about the block, including a timestamp of when the block was a created, a hash representing the address of the previous block, and a Merkle root hash (a cryptographic hash of all of the transactions included in the block) |
| block number | Blocks are numbered sequentially in chronological order of creation |
| hash | An output of a string of a fixed size derived from the transformation of data of any length. `tx.hash` is the hash of the transaction. It is generated by a one-way cryptographic algorithm, for which the original data cannot be retrieved by decryption. Hashing stores passwords and prevents fraudulent transactions and double spending in blockchain. |
| node | A programmed participant in the network that can interact with other nodes. Each node has different responsibilities, such as client or validator. |

Reply:
I totally agree. A general introduction to blockchain would be very nice to have, and it would define all these concepts.
At the same time, they could be added to the glossary.
But that's beyond the scope of this exercise, so don't worry.